### ISSUE TITLE
MotionPlan memory leaks

### ISSUE THREAD

**Issue by zlyxx0611** [krishauser/Klampt#98]
When I run the following code snipplet to do motion planning, there will be a bug _**'swig/python detected a memory leak of type 'std::string *', no destructor found.'**_

I located the problem:
1. only creating a CSpace, it's ok; 
2. just creating a motion planner (use either configurePlanner or simply planner = MotionPlan(...)）without using it will cause the bug           

This has been tested on both Klampt 0.8.3 & Klampt 0.8.6, Python3.5

**Note: I have already call CSpace.close() and motionplanning.destroy() when planning is done.**

Looking forward to your reply!

```python
  space = RigidObjectCSpace(group, collider=WorldCollider(world))
  # set start and goal
  start = space.transformToConfig((start_ori, start_pos))
  goal = space.transformToConfig((goal_ori, goal_pos))

  # check if goal is feasible
  goal_flag = True

  # make planner
  try:
      #planner, args = configurePlanner(space, start, goal, type='auto')
      planner = MotionPlan(space, type="sbl")#, connectionThreshold=0.2, shortcut=1)
      planner.setEndpoints(start, goal)
  except RuntimeError as err:
      print('Oops!!')
      goal_flag = False
....
  planner.close()
  motionplanning.destroy()
  space.close()
```

**Comment by krishauser** [comment]
Thanks for pointing this out! The warning is annoying, but you can probably ignore it as it is just a small memory leak and is relatively harmless. 

The problem was recently fixed in the "devel" branch. (The problem was a missing SWIG include, which requires recompiling the Python extension.) The fix will be made more widely available in the next pip release (0.8.7+) which is likely to come out in a month or so.

**Comment by krishauser** [comment]
Fixed in version 0.8.7
